README.md

```diff
@@ -2,12 +2,21 @@
 
 Pantheon CMD is a Python-based command-line tool that allows you to generate a rendered preview of modular documentation using the new HAML templates.
 
+Installing the Pantheon CMD tool using RPM allows users to perform calls using the predefined aliases such as: 
+* `pcmd validate`
+* `pcmd build`
+* `pcmd preview`
+
+Alternatively, a user can clone this repository and add the following `pcmd` scripts on $PATH, but the ability to use predefined aliases will not be possible.
+
 ## Directory Structure
 
 The top level of this repository contains the following files and directories:
 
 **build**
-A directory that contains the latest RPM for the script, and the SPEC file used to generate the RPM.
+A directory that contains the:
```

Comment:
s/A directory that contains the:/A directory that contains the following files:/